save-backend/Backend-API.md

```diff
@@ -0,0 +1,269 @@
+## SAVE Cloud Backend API
+### Overview
+
+The process of execution submission is very simple, all that you have to do
+is just to provide your credentials and information about your project, which you
+would like to evaluate and do few requests.\
+In this document will be represented the examples, which use `cURL`
+as instrument for working with SAVE Cloud API.
+
+To make the requests more universal, we will
+store major information, which is required by SAVE Cloud in requests, by
+into variables in **command prompt**.
+
+### Authorization configuration
+
+Each request, performed to API, require credentials, which could be configured like below, where
+
+* `SAVE_CLOUD_URL` - SAVE Cloud url and port
+* `SAVE_CLOUD_AUTH_SOURCE` where the identity is coming from, e.g. "github"
+* `SAVE_CLOUD_AUTH` - basic authorization header in format `source@username:token`.\
+   `source` is the same like `SAVE_CLOUD_AUTH_SOURCE`, and `token` is unique token,\
+    which could be created in personal settings in SAVE Cloud system.
+
+Just open up a command prompt and enter the following commands:
+
+```bash
+SAVE_CLOUD_URL=https://saveourtool.com:443
+
+SAVE_CLOUD_AUTH_SOURCE=github
+
+SAVE_CLOUD_AUTH='Basic Z2l0aHViQHVzZXJuYW1lOnRva2Vu'
+```
+
+### Configuration of information about evaluated tool
+
+To specify information about project, that you would like to evaluate,
+it's necessary to create the fields below in command promt. Only the `organizationName`,
+`projectName` and `gitUrl` are required, if your project is public:
+
+
+```bash
+# Required
+organizationName=Huawei
+
+# Required
+projectName='"save"'
```

Comment:
Why do we need several different quotes?

Reply:
We need to put such variables into `json` with quotes,
if we will assign them just like `projectName="save"` it will be interpret by bash like simple `save`, thus we need to surround it by additional `' '`, in aim to get `"save"`